src/main/resources/db/migration/V1_28__org_chart_updates.sql

```diff
@@ -0,0 +1,634 @@
+update locrefdata.jurisdiction set description = 'Employment Tribunals', last_update = timezone('utc', now())
+where jurisdiction_id = (select jurisdiction_id from locrefdata.jurisdiction where description = 'Employment Tribunerals');
+
+insert into jurisdiction (description, last_update)
+values
+    ('Upper Tribunal Tax and Chancery Chamber', timezone('utc', now())),
+    ('Upper Tribunal Lands Chamber', timezone('utc', now())),
+    ('Upper Tribunal Administrative Appeals Chamber', timezone('utc', now())),
+    ('Upper Tribunal Immigration and Asylum Chamber', timezone('utc', now())),
+    ('Employment Appeals', timezone('utc', now()));
+
+delete from locrefdata.service where service_code = 'BAA1' and service_description = 'Alternative Business Structures';
+
+delete from locrefdata.service where service_code = 'BAA3' and service_description = 'Claims Management Services';
+
+delete from locrefdata.service where service_code = 'BAA4' and service_description = 'Consumer Credit';
+
+update locrefdata.service set service_description = 'Gambling', last_update = timezone('utc', now())
+where service_code = 'BAA8' and service_description = 'Gambling Appeals';
+
+delete from locrefdata.service where service_code = 'BAB2' and service_description = 'Local Government Standards';
+
+insert into locrefdata.service (service_id, org_unit_id, business_area_id, sub_business_area_id, jurisdiction_id, service_code,
+  service_description, service_short_description, last_update) values (
+  46,
+  (select org_unit_id from locrefdata.org_unit where description='HMCTS'),
+  (select business_area_id from locrefdata.org_business_area where description ='Civil, Family and Tribunals'),
+  (select sub_business_area_id from locrefdata.org_sub_business_area   where description ='Tribunals'),
+  (select jurisdiction_id from locrefdata.jurisdiction where description ='General Regulatory Chamber'),
+  'BAB4',
+  'Community Right To Bid',
+  'Community Right To Bid',
+  timezone('utc', now()));
+
+insert into locrefdata.service (service_id, org_unit_id, business_area_id, sub_business_area_id, jurisdiction_id, service_code,
+  service_description, service_short_description, last_update) values (
+  47,
+  (select org_unit_id from locrefdata.org_unit where description='HMCTS'),
+  (select business_area_id from locrefdata.org_business_area where description ='Civil, Family and Tribunals'),
+  (select sub_business_area_id from locrefdata.org_sub_business_area   where description ='Tribunals'),
+  (select jurisdiction_id from locrefdata.jurisdiction where description ='General Regulatory Chamber'),
+  'BAB5',
+  'Electronic Comms etc Communications, Postal Services & Network Information Systems',
```

Comment:
I think this should be 'Electronic Communications, Postal Services & Network Information Systems'

Reply:
addressed